apis/src/components/organisms/dropdowns.rs

```diff
@@ -45,6 +46,7 @@ pub fn UserDropdown(username: String) -> impl IntoView {
             >
                 Config
             </a>
+            <DarkModeToggle/>
```

Comment:
Doesn't have the same height as the other buttons and therefore it looks a bit weird.